nomad/state/state_store.go

```diff
@@ -2265,8 +2265,13 @@ func (s *StateStore) CSIVolumesByPluginID(ws memdb.WatchSet, namespace, prefix,
 }
 
 // CSIVolumesByIDPrefix supports search. Caller should snapshot if it wants to
-// also denormalize the plugins.
+// also denormalize the plugins. If using a prefix, the results will not use an
+// index.
```

Comment:
Isn't this only true if we're also using the wildcard namespace?

Reply:
Ops, yeah, that's what I meant to write 😅 

Thanks!